src/tools/rbd/action/Bench.cc

```diff
@@ -207,6 +207,12 @@ int do_bench(librbd::Image& image, io_type_t io_type,
     return -EINVAL;
   }
 
+  int r = image.flush();
+  if (r < 0) {
+    std::cerr << "rbd: failed to flush: " << cpp_strerror(r) << std::endl;
+    return r;
```

Comment:
Do we want this to be fatal? E.g. previously running bench read against exclusively mapped image would succeed.

Reply:
Tweaked it to be fatal for write/read-write tests and skipped on the final flush for read-only tests